### ISSUE TITLE
Telegram widgets are blocked

### ISSUE THREAD

**Issue by theel0ja** [EFForg/privacybadger#2029]
![image](https://user-images.githubusercontent.com/5832930/40321771-d1c6f13c-5d38-11e8-9cc3-8e231a498073.png)

Demo: https://rawgit.com/theel0ja/e5c0c4f20a02478330be49b4fc6a2214/raw/ceabcfa2980fe7b818f18b0bc361e1f6ef6806da/telegram-widget-demo.html

**Comment by theel0ja** [comment]
When telegram.org is yellowlisted, t.me also needs it..

**Comment by theel0ja** [comment]
Script is on telegram.org, while iframe is on t.me

**Comment by ghostwords** [comment]
Hello! Could you run the following script in your Badger's background page console and share what it prints out?
```javascript
(function () {
  const STR = "t.me";
  console.log("**** ACTION_MAP for", STR);
  _.each(badger.storage.getBadgerStorageObject('action_map').getItemClones(), (obj, domain) => {
    if (domain.indexOf(STR) != -1) console.log(domain, JSON.stringify(obj, null, 2));
  });
  console.log("**** SNITCH_MAP for", STR);
  _.each(badger.storage.getBadgerStorageObject('snitch_map').getItemClones(), (sites, domain) => {
    if (domain.indexOf(STR) != -1) console.log(domain, JSON.stringify(sites, null, 2));
  });
}());
(function () {
  const STR = "telegram";
  console.log("**** ACTION_MAP for", STR);
  _.each(badger.storage.getBadgerStorageObject('action_map').getItemClones(), (obj, domain) => {
    if (domain.indexOf(STR) != -1) console.log(domain, JSON.stringify(obj, null, 2));
  });
  console.log("**** SNITCH_MAP for", STR);
  _.each(badger.storage.getBadgerStorageObject('snitch_map').getItemClones(), (sites, domain) => {
    if (domain.indexOf(STR) != -1) console.log(domain, JSON.stringify(sites, null, 2));
  });
}());
```
To get to the background page console in Chrome, visit `chrome://extensions`, make sure "Developer mode" is checked, click on the "background page" link in Privacy Badger's row, and select the Console tab.

In Firefox, visit `about:debugging`, enable add-on debugging, click Debug next to Privacy Badger, click the OK button on the popup warning about remote debugging, and enter the above script into the console after the >>.

**Comment by theel0ja** [comment]
```js
**** ACTION_MAP for t.me
t.me {
  "userAction": "user_cookieblock",
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1527114349653
}
jsonlint.memberful.com {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1519374144363
}
ystart.me {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1526605245303
}
bstart.me {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1526526025088
}
**** SNITCH_MAP for t.me
t.me [
  "localhost",
  "telegram.org",
  "theverge.com"
]
ystart.me [
  "start.me"
]
bstart.me [
  "start.me"
]
**** ACTION_MAP for telegram
telegram.org {
  "userAction": "user_cookieblock",
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1527054211585
}
telegram.me {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1518234353325
}
https-ru-telegram-store-com.disqus.com {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "cookieblock",
  "nextUpdateTime": 1519203462252
}
oauth.telegram.org {
  "userAction": "user_allow",
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1519298951032
}
**** SNITCH_MAP for telegram
telegram.org [
  "t.me",
  "telegram.me",
  "embed.ly",
  "jsbin.com"
]
telegram.me [
  "telegram.org"
]
```

**Comment by jhabdas** [comment]
<img width="475" alt="screen shot 2018-08-09 at 3 01 02 pm" src="https://user-images.githubusercontent.com/440298/43883242-13ba66d8-9be5-11e8-9d1d-2c465ddf9c5b.png">

If I were using privacy badger I'd want this blocked too given they seem to be doing redirection in order to collect google analytics data and hiding related code under base64 encoding. My two cents at least.

**Comment by ghostwords** [comment]
Fixed for now by 892b47ad42d840d1248d311ff7cad4545df43533 and 2a8414d80c0ec1c6ac52159d6442136ae6bfcd81.
